ecs-cli/integ/cfn/cfn.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2015-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+// Package cfn contains validation functions against a CloudFormation stack.
+package cfn
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestStackNameExists fails if there is no CloudFormation stack with the name stackName.
+func TestStackNameExists(t *testing.T, stackName string) {
+	client := newClient(t)
+	resp, err := client.DescribeStacks(&cloudformation.DescribeStacksInput{
+		StackName: aws.String(stackName),
+	})
+	if err != nil {
+		assert.FailNowf(t, "unexpected CloudFormation error during DescribeStacks", "wanted no errors, got %v", err)
+	}
+	if resp.Stacks == nil {
+		assert.FailNow(t, "stacks should not be nil")
```

Comment:
FFTI... but you can delete some lines if you do this:

```
import "github.com/stretchr/testify/require"

require.NoErrorf(t, err, "message", var)

// similar functions for NotNil, Equal, etc
```

Reply:
Thanks for sharing this! It made the code so much neater ✨